aiida/orm/utils/loaders.py

```diff
@@ -670,6 +704,55 @@ def _get_query_builder_label_identifier(cls, identifier, classes, operator='==',
         return builder
 
 
+class ContainerCodeEntityLoader(OrmEntityLoader):
+    """Loader for the `Code` entity and sub classes."""
+
+    @classproperty
+    def orm_base_class(self):
+        """
+        Return the orm base class to which loaded entities should be mapped. Actual queries to load an entity
+        may further narrow the query set by defining a more specific set of orm classes, as long as each of
+        those is a strict sub class of the orm base class.
+
+        :returns: the orm base class
+        """
+        from aiida.orm.nodes.data.container_code import ContainerCode
+        return ContainerCode
+
+    @classmethod
+    def _get_query_builder_label_identifier(cls, identifier, classes, operator='==', project='*'):
```

Comment:
Are you changing something in this method?

In a quick browse I didn't see anything specific to the container code (but I might be wrong).
You might be able to subclass `CodeEntityLoader`.

Reply:
As described above, if we simply make the entry point `core.code.containerized` then `load_code` can be used and none of this extra code is needed